### PR TITLE
Feature: ZEN-34363

### PR DESCRIPTION
**Pull request by zensaiello**

### CODE REVIEW

File: Products/ZenEvents/SyslogProcessing.py [hunk 1]
```diff
@@ -114,13 +41,30 @@ def __init__(self,sendEvent,minpriority,parsehost,monitor,defaultPriority):
...
         self.parsehost = parsehost
         self.sendEvent = sendEvent
         self.monitor = monitor
         self.defaultPriority = defaultPriority
-
+        self.compiledParsers = self.compiledParsers(syslogParsers)
+
+    def compiledParsers(self, parsers):
+        compiledParsers = []
+        for regex in parsers.split('\n'):
```
Comment (StevePC) [inline Products/ZenEvents/SyslogProcessing.py:56]: lets move to json so we can get tuples in the list of parsers

**Review by StevePC — CHANGES_REQUESTED** [review]
adjust config task to install a new SyslogProcessor (with new regexes) object on the main task during init

**Review by StevePC — APPROVED** [review]

File: Products/ZenEvents/SyslogProcessing.py (hunk 1 above)
Comment (zensaiello) [reply Products/ZenEvents/SyslogProcessing.py]: Config format updated to a JSON format.

File: Products/ZenEvents/SyslogProcessing.py [hunk 2]
```diff
@@ -103,9 +121,10 @@ def process(self, msg, ipaddr, host, rtime):
             #rest of msg now in summary of event
             evt = self.buildEventClassKey(evt)
             evt['monitor'] = self.monitor
-            evt['message'] = unicode(msg)
-            if 'summary' in evt:
-                evt['summary'] = unicode(evt['summary'] )
+            # Convert strings to unicode, previous code coverted 'summary' &
```
Comment (StevePC) [inline Products/ZenEvents/SyslogProcessing.py:124]: converted

**Review by StevePC — APPROVED** [review]

**Review by StevePC — APPROVED** [review]

**Comment by Deer-WarLord** [comment]
If we're applying a new configuration to the services, shouldn't that also be defined in the zenoss-service repo?

**Review by Deer-WarLord — APPROVED** [review]

**Review by StevePC — APPROVED** [review]

**Review by StevePC — COMMENTED** [review]
retest this

**Review by StevePC — APPROVED** [review]
retest

**Pull request merged**